Lesson Plans/Commercial/CPL.1 Commercial Overview, Preflight Preparation and Procedures, and Airport Operations.md

```diff
@@ -33,19 +33,48 @@ Ground 1.5-3 hours, depending on memory of concepts from [[PPL]]
 	2. **Operations/Systems**: Thorough understanding of systems and higher standards for operations
 	3. **Regulations**: Introduction of commercial operations requires a wider and deeper understanding of the [[FAR]]s
 	4. **Maneuvers**: [[PPL]] maneuvers are mostly 1 dimensional, [[CPL]] requires maneuvering simultaneously in 2 dimensions
-3. Preflight Preparation
-	1. May carry persons or property for hire subject to common carriage vs private carriage limitations
-		1. [[Private and Commercial Pilot Privileges and Limitation]] #todo :: Add source, maybe [[AC 61-142]]
-		2. [[Distinction Between Pilot Privileges and Operational Rules]] #todo :: Add source
-	2. [[CPL]]+[[IRA]] allows commercial flights >50nm and at night.
+3. [[Commercial ACS]] I. Preflight Preparation
+	1. [[Commercial ACS]] I.A. Pilot Qualifications
+		1. Builds on the foundation in [[Complete VFR Preflight Checklist]]/Pilot section
```

Comment:
Can do [[Complete VFR Preflight Checklist#Pilot to link direct]]

Reply:
Thanks!